src/bit_str.jl

```diff
@@ -1,127 +1,108 @@
-export BitStr, @bit_str, bcat, bit_literal, bit, to_location, onehot, onehot_batch
+export BitStr, @bit_str, @lbit_str, BitStr64, LongBitStr, bit_literal
+export bcat, onehot, onehot_batch, buf
```

Comment:
what is `buf` mean here? buffer? I think to follow our convention as usually we should use the full name? (and `buf` is too general which make it easily conflicts with user defined variables) or just don't export it by default then it would be pretty safe.

Reply:
changed to buffer.